### ISSUE TITLE
"This PDF document might not be displayed correctly." - Endless loading.

### ISSUE THREAD

**Issue by LukasReschke** [mozilla/pdf.js#6342]
The ownCloud Client Manual does not display in PDF.js and is endless loading, I do unfortunately not have enough insights in PDF.js to give more information (unless you request specific ones ;-)) but can share the PDF file in question: https://s3.owncloud.com/owncloud/index.php/s/FwAWNIYLOz6BTUd/download

![2015-08-11_12-39-22](https://cloud.githubusercontent.com/assets/878997/9195804/01099bee-4026-11e5-8816-9389cfe6b38f.png)

**Comment by timvandermeij** [comment]
Seems like another lexer issue as I found this in the console:

`Error: Invalid number: .`

**Comment by danielneis** [comment]
Hello,

I am getting the same problems with PDFs edited by FPDF and TCPDF libraries.
Altough the original PDFs created by LibreOffice works, when I edit them to add some text to every page, the resulting PDFs cannot be loaded, but I can open them with Evince and works fine.

**Comment by danielneis** [comment]
It seems to be that i was not sending the correct "Content-Length" header.

**Comment by Snuffleupagus** [comment]
This issue is most likely related to #3795.

**Comment by yurydelendik** [comment]
First page's XObject /Im1 contains corrupted content:
 
```
q
0.113725 0.176471 0.266667 rg /a0 gs
13.117 22.651 m 11.281 22.651 9.809 21.163 9.809 19.327 c 9.809 18.733 
9.961 18.174 10.234 17.69 c 11.34 18.315 12.621 18.678 13.98 18.678 c 
14.113 18.678 14.238 18.674 14.367 18.666 c 14.352 18.85 14.344 19.038 
14.344 19.229 c 14.344 20.252 14.566 21.225 14.957 22.1 c 14.43 22.455 
13.801 22.651 13.117 22.651 c h
13.117 22.651 m f
6.383 12.92 m 2.859 12.92 0 10.084 0 6.561 c 0 3.034 2.859 0.174 6.383 
0.174 c 7.727 0.174 8.969 0.592 9.996 1.299 c 9.57 1.959 9.32 2.748 
9.32 3.584.020.594 6. c 499 c08. c830.174 586.17 21.17436 
8.4 6.17436 9 c030.1717436 9 c 18.6.418.85930784 07.859 05.1717c08.
859 09.6.442.859 12m f
6.383 12651 m f
6.383 12.9f
678 13.757 5.651727563.757 5.7.0 2.8 c858.7.0 2.8 c030.177.0 2.09 8 
8.136.1778899 797 5.521265172496.17873.8 c90674 c95.65153174 c95.c.455 c 7.4 c95.651918.7.770.252105.7.74522.1 c047 61 18.67802 61623.67802 748069.229 c02 1 m 13.9 c719.651202 15c90678 c809.3215c195.654 18.6746 3.
117768.674469..75728.229 c 09..7578.6741452.757 5.678 13.757 5.651 c h 13.757 5.65f
252 36 344314..92616 1.344314.26114.94435.261419.944338.222614 12344972.4261135.344608.261135.344 19.22261135.361635.2614
9.961065.261320.234518.
2224493.661627 3171 0.654295.357 48.67868126533.0 2.8 c1 5.3 c079.038 4 2.3527563.52 32653 c879..75795.357 76.344314.261 36 344314.651 252 36 344314..9f
21.443.854 .65175714.854 .654014.2.3252.654014.344 19.22954014.37.776 
954522.66.44.66. 14.354314.65961 37.37.416 344746.809 13.178664.809 13.
2225409.809 13.16.443..34 97 234 4.37.841265234764.3522889234814.352252.
234814.344 19.22234814.2.3252.41
76814.46
21.443.854 .651 21.443.854 .6f
.356770.55 3 .653 7c08.55 3 5334225.651545.3514
9. c 0126533.042 1 8623
533c 09.08.4 6533c 09748069.2533c 0916.728.3527464c 013.96623252288.c. 3178 8.98 115336 21-2 0202.356770-2 0202.25972496-2 020242.4 2.54076 42.4 2683 12642.4 210.32 25972496.55 3 5356770.55 3 .651.356770.55 3 .69f
671531740732.651 598174073292 0497 5.418.2 04979.32 3.5.2 04979 c 22.41 59812 0 85.6515312 0 85.229548142 0 85..17979 022.667 11.34 15.c.45611359.328.674397 51918.6784 12m9928.229 48147404 5.677771740732f
671531740732.651 c 1531740732.69f
178664.175922.65809 4.175922.9 c470.250202.9 c470748069.229 c4708 c392f
809 4172713.178664172713.22261964172713.35 5868 c392.35 586748069.2
.35 586.2502022261964.175922
178664.175922.651 178664.175922.6f
Q
```

We shall probably ignore this object.

**Comment by Snuffleupagus** [comment]
Closing as fixed by PR #8240.
(But note that since the PDF file is corrupt, no PDF viewer is able to render the file completely.)
